<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HorizontalSpinBox</class>
 <widget class="QWidget" name="HorizontalSpinBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>99</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="downButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/libentertaining/icons/go-previous.svg</normaloff>:/libentertaining/icons/go-previous.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="upButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/libentertaining/icons/go-next.svg</normaloff>:/libentertaining/icons/go-next.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
